app/src/main/java/edu/card/clarity/presentation/Home.kt

```diff
@@ -1,28 +1,207 @@
 package edu.card.clarity.presentation
 
 import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 
 @Composable
 fun HomeScreen() {
-    Box(
+    Column(
         modifier = Modifier
             .fillMaxSize()
-            .background(Color.Magenta),
-        contentAlignment = Alignment.Center
+            .background(Color.White)
+            .padding(horizontal = 32.dp, vertical = 40.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+    ) {
+        Header(userName = "Sophie Chan")
```

Comment:
It would be more logical to combine the header and bar chart into a single composable representing the reward summary.